app/views/hours/show.html.slim

```diff
@@ -3,13 +3,18 @@
     .table-responsive
       table.table.table-no-border-top
         tbody
-          tr
-            - if current_user.superadmin?
+          - if policy(Hour).superadmin_or_volunteer_related?
```

Comment:
The point of this `if` was that only the superadmin "needs" to know every time which volunteer that concerns. The volunteer herself knows that the hours refer to her, thus she doesn't need to see her name. So either this should test if the user is superadmin (preferably through a policy) or it can be completely removed.